services/cypress/integration/cmsWorkflow/submissionReview.spec.ts

```diff
@@ -98,4 +99,74 @@ describe('CMS user can view submission', () => {
 
         })
     })
+
+    it('and can approve a submission via releasing it to the state', () => {
+        cy.interceptFeatureFlags({
+            'submission-approvals': true,
+            '438-attestation': true,
+            'hide-supporting-docs-page': true
+        })
+        // state user adds a new package
+        cy.logInAsStateUser()
+        cy.startNewContractAndRatesSubmission()
+        cy.fillOutBaseContractDetails()
+        cy.navigateContractForm('CONTINUE')
+
+        cy.findByRole('heading', {
+            level: 2,
+            name: /Rate details/,
+        }).should('exist')
+        cy.fillOutNewRateCertification()
+        cy.fillOutAdditionalActuaryContact()
+        cy.navigateContractRatesForm('CONTINUE')
+
+        cy.findByRole('heading', {
+            level: 2,
+            name: /Contacts/,
+        }).should('exist')
+        cy.fillOutStateContact()
+        cy.navigateContractForm('CONTINUE')
+
+        // store submission id for reference later
+        let submissionId = ''
+        cy.location().then((fullUrl) => {
+            const { pathname } = fullUrl
+            const pathnameArray = pathname.split('/')
+            submissionId = pathnameArray[2]
+        })
+
+        // submit package
+        cy.findByRole('heading', { level: 2, name: /Review and submit/ })
+        cy.submitStateSubmissionForm()
+
+        // store submission name for later
+        cy.location().then((loc) => {
```

Comment:
**Suggestion**: Not sure if there is a reason for manually creating and submitting the submission, but we have a custom command that can create a submitted contract and rates submission. That way we can speed up the test by skipping manually creating and submitting the submission. There is an example in `questionResponse.spec.ts`.